kernal/cbm/editor.s

```diff
@@ -261,13 +265,33 @@ loop3
 	and #(255-MODIFIER_4080)
 	sta shflag
 	
-	sec             ;Get current screen mode
-	jsr screen_mode
-
-	eor #SCREEN_MODE_4030
-	clc
-:	jsr screen_mode ;Set new screen mode, will clear screen
-
+;	sec             ;Get current screen mode
+;	jsr screen_mode
+;
+;	eor #SCREEN_MODE_4030
+;	clc
+;:	jsr screen_mode ;Set new screen mode, will clear screen
+	and #MODIFIER_SHIFT
+	bne scrpnc
+	jsr screen_toggle_default_nvram
+	bra doredy
+scrpnc
```

Comment:
If I understand this correctly, you can't get to Mode 3 (RGB) by pressing Shift+40/80. If Mode 3 is going to be supported, you could consider to iterate over all three modes.

Reply:
I now also tested this on hardware. 

I can confirm that pressing the 40/80 key works as expected (I think). 

I have no NTSC composite display, but at least the screen goes black when pressing Shift+40/80, and it comes back to VGA when pressing it again.

However, I could not get the computer to boot into NTSC mode. I tested this by writing the values in the @defaults table in screen.s:809 + a checksum (A7?) to NVRAM. I changed the third value in the table from 0x21 to 0x22. Maybe I did something wrong, it's fiddly. Did you get this to work on hardware yourself?

Reply:
I did not yet test on hardware but I was able to get the emu to boot in NTSC mode with an nvram file with these contents.
`00 00 22 80  80 00 00 A0  00 F0 61 06  21 20 20 00  00 A0 00 F0  61 6B`

Reply:
> If I understand this correctly, you can't get to Mode 3 (RGB) by pressing Shift+40/80. If Mode 3 is going to be supported, you could consider to iterate over all three modes.

Good point, I think it is best that we support every mode.